ngraph/src/ngraph/op/crop_and_resize.hpp

```diff
@@ -27,15 +27,16 @@ namespace ngraph
             class NGRAPH_API CropAndResize : public Op
             {
             public:
+                static constexpr NodeTypeInfo type_info{"CropAndResize", 0};
+                const NodeTypeInfo& get_type_info() const override { return type_info; }
```

Comment:
Should it be changed to `RTTI_DECLARATION`?

Reply:
See my another comment on that.